tree.go

```diff
@@ -834,7 +834,10 @@ func (n *InternalNode) GetProofItems(keys keylist) (*ProofElements, []byte, [][]
 			// repeated as many times? It would be wasteful, so the
 			// decoding code has to be aware of this corner case.
 			esses = append(esses, extStatusAbsentEmpty|((n.depth+1)<<3))
-			pe.Vals = append(pe.Vals, nil)
+			for _ = range group {
```

Comment:
Maybe `for i:=0; i< len(group); i++` is more natural and avoid the linter complaining?